### PR TITLE
Add Time duration support

### PR DESCRIPTION
**Pull request by radeno**
This PR contains:

- [ ] bugfix
- [x] feature
- [ ] refactor
- [ ] tests
- [ ] documentation
- [ ] metadata

### Breaking Changes?

- [ ] yes
- [x] no

### Please Describe Your Changes

This PR adds support for time duration values. Nothing more.

### CODE REVIEW

File: lib/nodes/Numeric.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const { registerWalker } = require('../walker');
 
 const Node = require('./Node');
 
-const unitRegex = /%|ch|cm|em|ex|in|mm|pc|pt|px|q|rem|vh|vmax|vmin|vw$/i;
+const unitRegex = /%|ch|cm|em|ex|in|mm|ms|pc|pt|px|s|q|rem|vh|vmax|vmin|vw$/i;
```
Comment (jonathantneal) [inline lib/nodes/Numeric.js]: As defined in:

CSS Level 2: https://www.w3.org/TR/CSS2/aural.html#times
CSS Values Level 3: https://drafts.csswg.org/css-values-3/#time
CSS Values Level 4: https://drafts.csswg.org/css-values-4/#time

**Comment by jonathantneal** [comment]
@shellscape, this seems pretty clean. I‘m not sure why the `unitRegex` has a `$` expression at the end of `vw`, but this change is consistent with how the tool already works.

**Comment by radeno** [comment]
I tried to add also angle units. But it doesn't take `90deg` as value, test fails. Another value, like `10deg` was ok. Don't have time to dig deeper why.
So support time values is small step forward.

**Comment by shellscape** [comment]
Thanks for putting this together 🍺

**Pull request merged**